spec/factories/ext_management_system.rb

```diff
@@ -258,12 +258,16 @@
   factory :ems_azure,
           :aliases => ["manageiq/providers/azure/cloud_manager"],
           :class   => "ManageIQ::Providers::Azure::CloudManager",
-          :parent  => :ems_cloud
+          :parent  => :ems_cloud do
+    provider_region { "eastus" }
```

Comment:
I believe this is a very flexible value in Openshift.  Should it be more random?  Maybe something like `Faker::Internet.slug`? https://github.com/stympy/faker/blob/master/doc/default/internet.md

Reply:
@bdunne have i ever told you what an excellent sense of humor you have, anyway?

Reply:
I mean, this is for Azure... does that matter?

Reply:
That looks like an AWS region, but I don't know that it matters?

Reply:
eastus is fine. The full list of valid regions is in the regions.rb file if you want to randomly select one of those.